### ISSUE TITLE
Cleft of alveolar ridge of maxilla

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9639]
**HPO term**
Cleft of alveolar ridge of maxilla -- keep this as primary label, merge with Cleft maxillary alveolus

change primary label to "Cleft maxillary alveolar ridge"

**Reason for deprecation**

**Suggested replacement or term to consider**

**Comment by pnrobinson** [comment]
@sabrinatoro 
Reannotating
- [ ] [OMIM:619452](https://hpo.jax.org/app/browse/disease/OMIM:619452)
Anencephaly 2
- [ ] [OMIM:268305](https://hpo.jax.org/app/browse/disease/OMIM:268305)
Robin sequence with cleft mandible and limb anomalies

**Comment by pnrobinson** [comment]
@sabrinatoro
merged

**Comment by sabrinatoro** [comment]
Thank you @pnrobinson !
